go.mod

```
@@ -36,6 +36,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver v0.33.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver v0.33.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/receivercreator v0.33.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.0.0-00010101000000-000000000000
```

we are pinning all collector components to the desired version
```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.33.0
```